components/mixins/experience-event/experienceevent-directmarketing.schema.json

```diff
@@ -11,6 +11,9 @@
   "type": "object",
   "meta:extensible": true,
   "meta:abstract": true,
+  "meta:tags": {
```

Comment:
@mjasrotia , @prabhum2 
Since this mixin only has the one change, is it okay to approve?